xtas/tests/test_pipeline.py

```diff
@@ -44,7 +44,7 @@ def test_pipeline():
     s = "cats are furry"
     expected = [('cats', 'NNS'), ('are', 'VBP'), ('furry', 'JJ')]
     result = pos_tag(tokenize(s), 'nltk')
-    assert_equal(result, expected)
```

Comment:
First a check without the pipeline, then with, so you can easily see whether the pipeline is at fault or not. Nice!